tests/test_hello_page.py

```diff
@@ -0,0 +1,32 @@
+"""Tests related to hello page"""
+import logging
+
+from constants import header
+
+
+class TestHelloPage:
+    """Stores tests for create hello page base functionality"""
+
+    log = logging.getLogger("[TestHelloPage]")
+
+    def sign_out_from_hello_page_after_sign_in(self, start_page, random_login):
```

Comment:
Is this test ? If so it should starts with 'test_'

Reply:
Fixed: rename test to "test_sign_out_from_profile_page_after_sign_in"